README.md

```diff
@@ -1 +1,18 @@
-# rekaz
\ No newline at end of file
+# Rikaz
+
+A Data Visualization App, shows the health status of the Palestinian Arabs in Israel.
+
+## Where is the data coming from?:
+
+[The Galilee Society](http://www.gal-soc.org): is an NGO dedicated to lower the gap and equality between  the Palestinian Arabs and the Jewish majority in Israel. One of the Galilee Departments is Rikaz, Rikaz is responsible to create and make data by surveys, through this App we aim to make Rikaz data more accessible to general public.  
+
+## How is the data visualized?:
+
+The main library that we use for data-visualization is [D3.js](https://d3js.org), also some of the charts are based on plain vanilla JS.
+
+For user interactions: **filtering** the data and filling the forms is done using the library [ReactJS](https://reactjs.org).  
+
+## Have a look
+
+
+[![The prototype](rikaz.png)](https://www.figma.com/proto/IeBFPegDJoYORSqJFwHXRKgc/Untitled?scaling=contain&node-id=4%3A2)
```

Comment:
You shouldn't have pictures only used in your README in your file structure as they aren't necessary. Post the picture in an issue on Github and then get the link to that picture.

Reply:
Moved, certainly we will add more to the readme, I rushed since we don't wont to have empty readme :smile: